diff --git a/build/buildflag.h b/build/buildflag.h
index 634697986c..24fec974b2 100644
--- a/build/buildflag.h
+++ b/build/buildflag.h
@@ -42,6 +42,13 @@
 // Because the flag is accessed as a preprocessor macro with (), an error
 // will be thrown if the proper header defining the internal flag value has
 // not been included.
+
+#if defined(ENTERPRISE_BROWSER)
+#define BUILDFLAG_INTERNAL_IS_ENTERPRISE_BROWSER() (1)
+#else
+#define BUILDFLAG_INTERNAL_IS_ENTERPRISE_BROWSER() (0)
+#endif
+
 #define BUILDFLAG(flag) (BUILDFLAG_CAT(BUILDFLAG_INTERNAL_, flag)())
 
 #endif  // BUILD_BUILDFLAG_H_
