diff --git a/chrome/browser/ui/views/profiles/profile_picker_view.cc b/chrome/browser/ui/views/profiles/profile_picker_view.cc
index e7d1b55897..2215389068 100644
--- a/chrome/browser/ui/views/profiles/profile_picker_view.cc
+++ b/chrome/browser/ui/views/profiles/profile_picker_view.cc
@@ -233,6 +233,15 @@ void ProfilePicker::CancelSignInFlow() {
   }
 }
 
+#if BUILDFLAG(IS_ENTERPRISE_BROWSER)
+// static
+void ProfilePicker::NotifyFirstRunAbandoned() {
+  if (g_profile_picker_view && g_profile_picker_view->flow_controller_) {
+    g_profile_picker_view->flow_controller_->OnFirstRunAbandoned();
+  }
+}
+#endif
+
 // static
 base::FilePath ProfilePicker::GetPickerProfilePath() {
   return ProfileManager::GetSystemProfilePath();
@@ -417,8 +426,10 @@ void ProfilePickerView::Clear() {
 }
 
 bool ProfilePickerView::ShouldUseDarkColors() const {
-  return GetNativeTheme()->preferred_color_scheme() ==
-         ui::NativeTheme::PreferredColorScheme::kDark;
+  return false;
+  // TODO(EBCM) support themes properly
+  // return GetNativeTheme()->preferred_color_scheme() ==
+  //        ui::NativeTheme::PreferredColorScheme::kDark;
 }
 
 content::WebContents* ProfilePickerView::GetPickerContents() const {
